service/geopmdpy/runtime.py

```diff
@@ -181,7 +227,7 @@ def update(self, signals):
         raise NotImplementedError('Agent is an abstract base class')
 
     def get_period(self):
-        """Get the target time interal for the control loop
+        """Get the target time internal for the control loop
```

Comment:
```suggestion
        """Get the internal target time for the control loop
```
or just
```suggestion
        """Get the target time for the control loop
```

It's a getter.  It's assumed it's returning internal state.

Reply:
I think that was supposed to be *interval* not internal